Tasks/PublishSymbols/IndexHelpers/DbghelpFunctions.ps1

```diff
@@ -1,26 +1,30 @@
+function Get-DbghelpPath {
+    [CmdletBinding()]
+    param()
+
+    Trace-VstsEnteringInvocation $MyInvocation -Parameter @()
+    $dbghelpPath = [System.IO.Path]::GetFullPath("$PSScriptRoot\..\dbghelp.dll")
+    Assert-VstsPath -LiteralPath $dbghelpPath -PathType Leaf -PassThru
```

Comment:
Thanks, will omit the -PassThru parameters on the Assert-VstsPath calls throughout my commit. Didn't know about the tracing already there and will simplify this.